deep_quoridor/src/agents/core/agent.py

```diff
@@ -11,15 +11,25 @@ def _friendly_name(class_name: str):
     def name(self) -> str:
         return Agent._friendly_name(self.__class__.__name__)
 
-    def get_action(self, game) -> int:
-        raise NotImplementedError("You must implement the get_action method")
+    def is_trainable(self) -> bool:
+        """Returns True if the agent is a learning agent, False otherwise."""
+        return False
+
+    def start_game(self, game, player_id):
```

Comment:
Nice, actually I was working on an agent and noticed that the agent didn't know if it's player 1 or 2, so it doesn't explicitly know where to go.  I was adding it in the reset method but this works as well.  I think in the future we may want to standarize all this to use Player.ONE from quoridor.py, but we can do that separately